bookkeeper-stats-providers/prometheus-metrics-provider/src/test/java/org/apache/bookkeeper/stats/prometheus/TestPrometheusMetricsProvider.java

```diff
@@ -70,9 +69,13 @@ public void testStartNoHttpWhenBkHttpEnabled() {
         PropertiesConfiguration config = new PropertiesConfiguration();
         config.setProperty(PrometheusMetricsProvider.PROMETHEUS_STATS_HTTP_ENABLE, true);
         config.setProperty("httpServerEnabled", true);
-        @Cleanup("stop") PrometheusMetricsProvider provider = new PrometheusMetricsProvider();
-        provider.start(config);
-        assertNull(provider.server);
+        PrometheusMetricsProvider provider = new PrometheusMetricsProvider();
```

Comment:
Well, It was actually related to PR. Because lombok was being used just once in the entire test module so initial thought was instead of pulling lombok as test dependency, test can be modified not to use it. But I think that was a bad idea. I am going to undo this change and pull lombok in the test dependency.

Reply:
So far pulgin configuration is not so evolved. If we need to configure plugin we can do similar to what is being done in Pravega.

Reply:
@fpj - wondering if we could follow up for Gradle plugin configurations in a follow up PR?